index.js

```diff
@@ -48,6 +48,7 @@ class KeyringController extends EventEmitter {
 
     this.encryptor = opts.encryptor || encryptor;
     this.keyrings = [];
+    this.unsupported_keyrings = [];
```

Comment:
Should we use `camelCase` for this property name?

Reply:
Done